src/backend/distributed/utils/statistics_collection.c

```diff
@@ -95,8 +101,11 @@ CollectBasicUsageStatistics(void)
 	escape_json(fields, unameData.release);
 	appendStringInfoString(fields, ",\"hwid\": ");
 	escape_json(fields, unameData.machine);
+	appendStringInfo(fields, ",\"node_metadata\": %s", metadataJsonbStr);
```

Comment:
I couldn't find a better API to construct json objects in C code. `json_build_object` seemed to be an option, but calling it inside C code is not very simple.

Reply:
Yeah that's fine.